profiles/common/modules/custom/multisite_drupal_toolbox/multisite_drupal_toolbox.module

```diff
@@ -1657,3 +1656,54 @@ function multisite_drupal_toolbox_features_post_restore($op, $items) {
     }
   }
 }
+
+/**
+ * Remove PHP Warning logs from HTML5 during a loadHTML.
+ *
+ * Extend SimpleXMLElement function.
```

Comment:
It rather enhances the behaviour of DOMDocument::LoadHTML(), not SimpleXMLElement.